packages/webapp/public/locales/pt/translation.json

```diff
@@ -856,7 +856,12 @@
       "AMPHIBIANS": "Anfíbios",
       "BIRDS": "Aves",
       "CROP_VARIETIES": "Variedades de cultivos",
-      "ERROR": {
+      "API_OFFSET_LIMIT_ERROR": {
+        "BODY": "MISSING",
+        "PREVIEW": "Unavailable",
+        "TITLE": "MISSING"
```

Comment:
I think you forgot to add the title "Tinha um problema" as you did for the other languages.

Reply:
Good call! 🙂